<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddRooms</class>
 <widget class="QWidget" name="AddRooms">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add New Room</string>
  </property>
  <property name="windowIcon">
   <iconset resource="bill_master_resources.qrc">
    <normaloff>:/icons/icons/1428288011_10.png</normaloff>:/icons/icons/1428288011_10.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="styleSheet">
      <string notr="true">background:#fff;</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="roomDetailsTab">
      <attribute name="title">
       <string>Basic</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Room Details</string>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>481</width>
            <height>22</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Room Number</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="roomNumber"/>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Room Type</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="roomType"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>60</y>
            <width>481</width>
            <height>111</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Room Description</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="roomDescription"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>80</height>
          </size>
         </property>
         <property name="title">
          <string>Beds Details</string>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>471</width>
            <height>31</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Single Beds</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="singleBeds"/>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Double Beds</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QSpinBox" name="doubleBeds"/>
           </item>
           <item row="0" column="4">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Extra Beds</string>
             </property>
            </widget>
           </item>
           <item row="0" column="5">
            <widget class="QSpinBox" name="extraBed"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Room Information</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>Room Equipments</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QListWidget" name="equipments">
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::MultiSelection</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_4">
              <property name="title">
               <string>Room Rates</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <layout class="QGridLayout" name="gridLayout_3">
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_7">
                   <property name="text">
                    <string>Rate per room</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLineEdit" name="rate_per_room"/>
                 </item>
                 <item row="0" column="2">
                  <widget class="QLabel" name="label_8">
                   <property name="text">
                    <string>Rate per person</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="3">
                  <widget class="QLineEdit" name="rate_per_person"/>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_5">
              <property name="title">
               <string>Room Status</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <layout class="QGridLayout" name="gridLayout_4">
                 <item row="0" column="0">
                  <widget class="QRadioButton" name="room_status_none">
                   <property name="text">
                    <string>None</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QRadioButton" name="room_status_need_cleaning">
                   <property name="text">
                    <string>Needs Cleaning</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="2">
                  <widget class="QRadioButton" name="room_status_clean">
                   <property name="text">
                    <string>Clean</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="3">
                  <widget class="QRadioButton" name="room_status_clean_checked">
                   <property name="text">
                    <string>Clean Checked</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_6">
              <property name="title">
               <string>GST Details</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_7">
               <item>
                <layout class="QGridLayout" name="gridLayout_2">
                 <item row="0" column="5">
                  <widget class="QLabel" name="label_13">
                   <property name="text">
                    <string>%</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="3">
                  <widget class="QLabel" name="label_10">
                   <property name="text">
                    <string>sGST</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="6">
                  <widget class="QLabel" name="label_11">
                   <property name="text">
                    <string>GST</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_9">
                   <property name="text">
                    <string>cGST</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QDoubleSpinBox" name="cGst">
                   <property name="alignment">
                    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                   </property>
                   <property name="buttonSymbols">
                    <enum>QAbstractSpinBox::NoButtons</enum>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="7">
                  <widget class="QDoubleSpinBox" name="gst">
                   <property name="alignment">
                    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                   </property>
                   <property name="readOnly">
                    <bool>true</bool>
                   </property>
                   <property name="buttonSymbols">
                    <enum>QAbstractSpinBox::NoButtons</enum>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="4">
                  <widget class="QDoubleSpinBox" name="sGst">
                   <property name="alignment">
                    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                   </property>
                   <property name="buttonSymbols">
                    <enum>QAbstractSpinBox::NoButtons</enum>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="2">
                  <widget class="QLabel" name="label_12">
                   <property name="text">
                    <string>%</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="8">
                  <widget class="QLabel" name="label_14">
                   <property name="text">
                    <string>%</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="saveRoom">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelAddNewRoom">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>roomNumber</tabstop>
  <tabstop>roomType</tabstop>
  <tabstop>roomDescription</tabstop>
  <tabstop>singleBeds</tabstop>
  <tabstop>doubleBeds</tabstop>
  <tabstop>extraBed</tabstop>
  <tabstop>equipments</tabstop>
  <tabstop>rate_per_room</tabstop>
  <tabstop>rate_per_person</tabstop>
  <tabstop>room_status_none</tabstop>
  <tabstop>room_status_need_cleaning</tabstop>
  <tabstop>room_status_clean</tabstop>
  <tabstop>room_status_clean_checked</tabstop>
  <tabstop>cGst</tabstop>
  <tabstop>sGst</tabstop>
  <tabstop>gst</tabstop>
  <tabstop>saveRoom</tabstop>
  <tabstop>cancelAddNewRoom</tabstop>
 </tabstops>
 <resources>
  <include location="bill_master_resources.qrc"/>
 </resources>
 <connections/>
</ui>
